dll/ntdll/ldr/ldrinit.c

```diff
@@ -1875,8 +1876,15 @@ LdrpInitializeProcess(IN PCONTEXT Context,
     Status = RtlAllocateActivationContextStack(&Teb->ActivationContextStackPointer);
     if (!NT_SUCCESS(Status)) return Status;
 
-    // FIXME: Loader private heap is missing
-    //DPRINT1("Loader private heap is missing\n");
+    RtlZeroMemory(&HeapParameters, sizeof(HeapParameters));
+    HeapFlags = HEAP_GROWABLE | HEAP_CLASS_1;
+    HeapParameters.Length = sizeof(RTL_HEAP_PARAMETERS);
+    LdrpHeap = RtlCreateHeap(HeapFlags, 0, 0x10000, 6000, 0, &HeapParameters);
```

Comment:
6000? Shouldn't CommitSize be divisible by PAGE_SIZE?

Reply:
Can we use some #define'd values there?